tests/test_khavee.py

```diff
@@ -12,8 +12,20 @@ class TestKhaveePackage(unittest.TestCase):
     def test_check_sara(self):
         self.assertEqual(kv.check_sara("เริง"), "เออ")
 
-    def test_check_marttra(self):
+    def test_check_marttra(self):    
+        self.assertEqual(kv.check_marttra("ปลิง"), "กง") 
+        self.assertEqual(kv.check_marttra("ลม"), "กม")   
+        self.assertEqual(kv.check_marttra("โปรย"), "เกย")
         self.assertEqual(kv.check_marttra("สาว"), "เกอว")
+        self.assertEqual(kv.check_marttra("บก"), "กก")
+        self.assertEqual(kv.check_marttra("โรค"), "กก")
+        self.assertEqual(kv.check_marttra("จด"), "กด")
+        self.assertEqual(kv.check_marttra("ตรวจ"), "กด")
+        self.assertEqual(kv.check_marttra("คน"), "กน")
+        self.assertEqual(kv.check_marttra("ทมิฬ"), "กน")
```

Comment:
```suggestion
        self.assertEqual(kv.check_marttra("ทมิฬ"), "กน")
        self.assertEqual(kv.check_marttra("บรร"), "กน")
        self.assertEqual(kv.check_marttra("กร"), "กน")
```

Can we take the opportunities to add these tests as well please? Thank you.

Reply:
My thought was to add 1 for direct cases (the end character matched the mattra) and 1 of the more complex case where's the character doesn't to kept it consistant but adding more wouldn't hurt.